Source/VMware.vSphereDSC/Tests/Integration/VMHostVss.Integration.Tests.ps1

```diff
@@ -84,29 +84,15 @@ $script:mofFileWithRemoveVss = "$script:integrationTestsFolderPath\$($script:con
 try {
     Describe "$($script:dscResourceName)_Integration" {
         Context "When using configuration $($script:configWithNewVss)" {
```

Comment:
The BeforeEach clause is missing here and should be added everywhere it is missing.

Reply:
Changed